File: sdf/sdf/ViewController.m
//
//  ViewController.m
//  sdf
//
//  Created by 张凯泽 on 15/9/24.
//  Copyright © 2015年 rytong. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSString *string = @"egbank://com.egbank.mobile?id=samli";
    [[UIApplication sharedApplication]openURL: [NSURL URLWithString:string ]];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Swift_for循环/Swift_for循环/ViewController.swift
//
//  ViewController.swift
//  Swift_for循环
//
//  Created by 张凯泽 on 15/9/28.
//  Copyright © 2015年 rytong. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        //第一种for 循环
        for var index : Int = 0 ;index<5 ; index++
        {
            //print(index)
        }
        //第二种for循环
        for var index = 0 ; index<5 ; index++
        {
            print(index);
        }
        //使用for。。。in。。。
        let a  = 1...5
        for temp in a
        {
           print(temp)
        }
        
        
    }

    
}

